content/en/docs/reference/command-line-tools-reference/feature-gates/structured-authorization-configuration.md

```diff
@@ -6,13 +6,16 @@ _build:
   render: false
 
 stages:
-  - stage: alpha 
+  - stage: alpha
     defaultValue: false
     fromVersion: "1.29"
     toVersion: "1.29"
-  - stage: beta 
+  - stage: beta
     defaultValue: true
     fromVersion: "1.30"
+  - stage: stable
```

Comment:
`toVersion` should be added to the previous stage.

```suggestion
    toVersion: "1.31"
  - stage: stable
```

Reply:
done